[[ bug 16144 ]] Dictionary title bar showing off screen on opening

runrevelanor opened the pull request:
Changed the revIDEEnsureOnScreen handler to check the effective top of the stack.

runrevelanor committed:
[[ bug 16144 ]] Dictionary title bar showing off screen on opening

Changed the revIDEEnsureOnScreen handler to check the effective top of the stack.

livecodeali commented:
What's the 30 here? Should it be (the top of pStackName - the effective top of stack pStackName)?

runrevelanor committed:
[[ bug 16144 ]] Title bar of Dictionary can appear off screen on opening

Updated to use the difference between the top and effective top of the stack to reset the position.

livecodeali commented:
@livecode-vulcan review ok  48d656c

livecode-vulcan commented:
💙 review by @livecodeali ok 48d656c21c78367fc8c35cf3e5301f5c894d779b
